strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-composer-layout-provider/src/test/java/org/carlspring/strongbox/artifact/coordinates/ComposerArtifactCoordinatesTest.java

```diff
@@ -0,0 +1,96 @@
+package org.carlspring.strongbox.artifact.coordinates;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import edu.emory.mathcs.backport.java.util.Arrays;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.data.jdbc.DataJdbcTest;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+/**
+ * Tests for ComposerArtifactCoordinates class
+ *
+ * @author jcoelho
+ */
+public class ComposerArtifactCoordinatesTest
+{
+
+    private static List<String> packages;
+
+    static
+    {
+        packages = new ArrayList<>();
+    }
+
+    @BeforeAll
+    static void setup()
+    {
+
```

Comment:
Cosmetic: Too many empty lines all over. It makes sens to put an empty line before and after a logical block to make it stand out, but in this case, it looks like it's a bit too much. Would you mind removing these (here and in the other method)? Thanks! :)

Reply:
Fixed